workflows-scripts/marc-authority/view-marc-authority/README.md

```diff
@@ -0,0 +1,45 @@
+## Workflow steps:
+##### Workflow for marc_viewAuthorityRecord.jmx
+
+View MARC Authority Record:
+
+1. Go to MARC Authority app 
+2. In the Search & filter left pane, do a Keyword search (Using the Search option)
+3. Results will appear in the middle pane.
+4. Click on any result in the result list to view the detail authority record
+Repeat step 4 by clicking on other results in the result list to bring up other MARC authority records to view.
```

Comment:
Will the JMeter script repeat step 4 indefinitely, or for a fixed number of loops, or for some duration before repeating all the steps? I think the majority of the time this script should be spending time in step 4.

Reply:
The step 4 will be executed randomly from 1 to 3 times with random record id where authRefType=='Authorized'